faunadb-android/src/test/java/com/faunadb/client/AndroidClientSpec.java

```diff
@@ -220,9 +221,15 @@ public void shouldTestNestedReferences() throws Exception {
 
     Expr allNestedClasses = Classes(nestedDatabase);
 
+    List<RefV> results = new ArrayList();
```

Comment:
`Arrays.asList(...)`?

Reply:
I like the way this style keeps line length low. There are only three elements... Either works.

Reply:
Heh. And I broke this test...

Reply:
Don't you need the diamond thing? `<>`

Reply:
Huh. I didn't realize that would compile... I suppose I don't anymore? I'm not up to speed on all of the Java 8 changes.